### ISSUE TITLE
Overlapping numbers

### ISSUE THREAD

**Issue by moofawsaw** [IonDen/ion.rangeSlider#635]
Though still dynamic with a long width, when numbers go into the 1000's they overlap at a certain size.

[https://imgur.com/a/166D00v](url)

**Comment by IonDen** [comment]
Yeah, too many numbers could clash some times, this problem is under investigation.
